docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -67,7 +71,7 @@ WARNING: Listening on any other interface than the local loopback poses a securi
 
 * `path`: The URL of the endpoint.
 
-When starting a new server within a component, {prod-short} autodetects this, and the UI offers to automatically expose this port as an `exposure` port. This exposure is useful for debugging a web application. It is impossible to do this for servers that autostart with the container, for example, a database server. For such components, specify the endpoints explicitly.
+Devfiles autodetect a new server starting within a component, and the UI offers to automatically expose this port as an `exposure` port. Use this port to debug a web application. You cannot debug a web application for servers that autostart with the container, such as a database server. For such components, specify the endpoints explicitly.
```

Comment:
Again, this is about Eclipse Che. Devfiles have no UI! This needs to be rewritten to be implementation-agnostic.

Reply:
@jc-berger, not resolved.

Reply:
Since devfiles has no UI and we're not writing about Eclipse Che anymore, I'm thinking this entire section could be removed. @maysunfaisal what do you think?